### ISSUE TITLE
Ankhegs sometimes stop being green

### ISSUE THREAD

**Issue by lynxlynxlynx** [gemrb/gemrb#1591]
No particular pattern seen.
![image](https://user-images.githubusercontent.com/121515/157770532-3c3562f0-331e-430f-835d-a4a7b76a9146.png)
ankheg01.cre

**Comment by lynxlynxlynx** [comment]
As expected this happens regardless of sdl branch, opengl. Since it's visible with the same animations (same cycle and stance), we perhaps do something bad to the palettes. But it also reverts back to green now and then, so it's not a destructive/permanent change. ASAN doesn't report anything funny either, so I wonder what's causing the fluctuations.

**Comment by MarcelHB** [comment]
Hm, maybe also the exception case mentioned here?
https://github.com/gemrb/gemrb/blob/9122d29f65406a24ba2044dbbd58bfdd20a7f2f1/gemrb/core/CharAnimations.cpp#L290-L294

**Comment by lynxlynxlynx** [comment]
That's probably it, since if you look at `CharAnimations::GetPartPalette`, we deliberately leave it alone for one part. Ankheg animations are basically split between the mud and creature itself.

**Comment by lynxlynxlynx** [comment]
Yeah, disabling it for this animation type makes the problem almost go away completely. There's still a frame or two that's off, but it's also from a different stance, since the non-continuity of the animation is more obvious than the short color degradation.
